schemas/README.md

```diff
@@ -94,6 +120,120 @@ name | value | description
 
 
 
+## enum SceneEntityDeletionType
+
+An enumeration indicating which entities should match a SceneEntityDeletion command
+
+name | value | description
+---- | ----- | -----------
+`MATCHING_ID` | 0 | 
+`ALL` | 1 | 
+
+
+
+## ArrowPrimitive
+
+A primitive representing an arrow
+
+<table>
+  <tr>
+    <th>field</th>
+    <th>type</th>
+    <th>description</th>
+  </tr>
+<tr>
+<td><code>pose</code></td>
+<td>
+
+[Pose](#pose)
+
+</td>
+<td>
+
+Position of the arrow's tail and orientation of the arrow. Identity orientation means the arrow points in the +x direction.
+
+</td>
+</tr>
+<tr>
+<td><code>length</code></td>
```

Comment:
why not `shaft_length`, to match `shaft_diameter`, `head_length`, `head_diameter`?

Reply:
We could do that. The tradeoff is that the total length (`shaft_length+head_length`) is not included in the message anywhere.